### PR TITLE
LPS-128658 Fix misalignment of icons in the portlet-topper-toolbar

### PR DESCRIPTION
**Pull request by kresimir-coko**
Fixes [LPS-128658](https://issues.liferay.com/browse/LPS-128658) 

I've taken the CSS route fixing this, adding a new rule that targets `.portlet-topper-toolbar .component-action`. The rule reduces the `height` and `width` from the default `2rem` to `1rem` and changes the display to `inline-flex` allowing me to easily center the icons horizontally inside their containers.

I'm very unfamiliar with how we handle these changes, which theme to change, is a change like this the prefered way (or applying a certain class to the element).

### CODE REVIEW

**Review by julien — APPROVED** [review]
Tested and looks good to me.

**Before this change**
before

**With this change**
after

**Comment by julien** [comment]
cc @marcoscv-work @edalgrin

File: modules/apps/frontend-theme/frontend-theme-classic/src/css/portlet/_portlet.scss [hunk 1]
```diff
@@ -55,6 +55,13 @@
 			width: 1.5rem;
 		}
 
+		.portlet-topper-toolbar .component-action {
+			display: inline-flex;
```
Comment (pat270) [inline modules/apps/frontend-theme/frontend-theme-classic/src/css/portlet/_portlet.scss:59]: We don't need `inline-flex` here, already covered by `component-action`

**Review by pat270 — COMMENTED** [review]
We could replace some code in styled theme https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-theme/frontend-theme-styled/src/main/resources/META-INF/resources/_styled/css/portlet/_topper.scss#L105-L116 to:

```
.component-action {
	background-color: transparent;
	color: $portlet-topper-link-color;
}
```

**Review by pat270 — COMMENTED** [review]
Not sure whats going on github not saving my comments.

We don't need `inline-flex` or `justify-content`, already covered by `component-action`. Also using `1.5rem` for `width` and `height` would make the click area larger.

We could remove https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-theme/frontend-theme-classic/src/css/portlet/_portlet.scss#L51-L61 and add `font-size: 0.875rem` to `component-action` and it should size the icon.

**Comment by kresimir-coko** [comment]
Hey @pat270 I've experimented a bit trying the alternative solution you suggested and here are results.

### Using `width` and `height`
image

### Removing the 2 rules and using `font-size`
image

I decided to stick with just overriding the `width` and `height` as it retains the original dimensions of the topper.
I've force-pushed a commit (https://github.com/liferay-frontend/liferay-portal/pull/897/commits/3360f363416e7e6d41b42aa84ea769212f307dc2), overriding the old one so we don't get it confused as it's all in the same file.

**Comment by kresimir-coko** [comment]
ci:test:sf

**Comment by kresimir-coko** [comment]
ci:test:relevant

**Review by edalgrin — APPROVED** [review]
I ~should~ would forward this since it fixes the bug.

About other changes and optimizations we might want to open a subtask to review and eventually refactor/improve the whole portlet/portlet-topper

**Comment by pat270** [comment]
@kresimir-coko the reason for the gap: You need to change width and height on `component-action`. It's 2rem (32px) by default. If you overwrite it to `1.5rem` it will look better. The gaps aren't related to the font-size change, you can see the icons are the same size in both screenshots.

**Comment by kresimir-coko** [comment]
Thanks @pat270 I didn't investigate further as I just wanted to fix the bug.

**Comment by jbalsas** [comment]
Hey @edalgrin, @marcoscv-work, @pat270, is this forwardable or is there something else that needs to be done? 
- If it's the first, please make sure to hit the `ci:forward` to avoid others from having to reason about the state of the PR.
- If it isn't, please be explicit and clearly state what you expect from the one that sent the PR so it doesn't stay in limbo forever

@kresimir-coko, also up to you to ask for clarification to make sure this is forwardable or if it needs something else on your side! ;)

**Comment by marcoscv-work** [comment]
It works for me, I bet for the simplest solution and go ahead!

**Comment by kresimir-coko** [comment]
ci:forward

**Pull request closed without merging**